build/webpack/webpack.config.base.js

```diff
@@ -143,7 +143,9 @@ if ((globalThis.process || binding.process).argv.includes("--profile-electron-in
             transpileOnly: onlyPrintingGraph,
             ignoreDiagnostics: [
               // File '{0}' is not under 'rootDir' '{1}'.
-              6059
+              6059,
+              // Private field '{0}' must be declared in an enclosing class.
```

Comment:
This trips on [at least this line from Node](https://github.com/nodejs/node/blob/ee1ec50c108f139ac5e2f91f7719b4c8cfa74b96/lib/internal/url.js#L353) - figured it was easier to dd an ignore than try to patch it or otherwise may `tsc` happy.